fieldservice_survey/controllers/main.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2023 - TODAY, Marcel Savegnago <marcel.savegnago@escodoo.com.br>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, http
```

Comment:
```suggestion
from odoo import http
```

Get rid of this in favor of `request.env._`